march_hardware/include/march_hardware/MarchRobot.h

```diff
@@ -40,7 +40,7 @@ class MarchRobot
   MarchRobot(MarchRobot&&) = delete;
   MarchRobot& operator=(MarchRobot&&) = delete;
 
-  void resetIMC();
+  void resetImotionCubes();
```

Comment:
```suggestion
  void resetIMotionCubes();
```
iMotionCube is written with capital M everywhere else :sweat_smile: sorry